libs/community/pyproject.toml

```diff
@@ -155,6 +156,7 @@ tiktoken = ">=0.3.2,<0.6.0"
 anthropic = "^0.3.11"
 langchain-core = { path = "../core", develop = true }
 fireworks-ai = "^0.9.0"
+premai = "^0.3.25"
```

Comment:
My reading of the instructions for the `[tool.poetry.group.test_integration.dependencies]` group above is that we should not add `premai` here.

note that CI is failing because the poetry lock file is out of sync with this file:
```
Error: poetry.lock is not consistent with pyproject.toml. Run `poetry lock [--no-update]` to fix it.
```

Reply:
> My reading of the instructions for the `[tool.poetry.group.test_integration.dependencies]` group above is that we should not add `premai` here.
> 
> note that CI is failing because the poetry lock file is out of sync with this file:
> 
> ```
> Error: poetry.lock is not consistent with pyproject.toml. Run `poetry lock [--no-update]` to fix it.
> ```

As far I have see, not adding premai was failing the tests. So I added that. But okay lemme remove it on toml and do a lock update and see, what happens.